### PR TITLE
Fixes test description + small fix follow up

### PR DESCRIPTION
**Pull request by AlexOn1ine**
Title

### CODE REVIEW

File: test/battle/ability/emergency_exit.c [hunk 1]
```diff
@@ -36,3 +36,22 @@ SINGLE_BATTLE_TEST("Emergency Exit switches out when taking 50% max-hp damage af
...
+{
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET)
+        OPPONENT(SPECIES_GOLISOPOD) { Ability(ABILITY_EMERGENCY_EXIT); MaxHP(263); HP(133); Item(ITEM_ORAN_BERRY); };
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN {
+            MOVE(player, MOVE_SUPER_FANG);
+            SEND_OUT(opponent, 1);
+        }
```
Comment (Bassoonian) [inline test/battle/ability/emergency_exit.c:50]: Don't we generally keep each turn on a single line?

Comment (AlexOn1ine) [reply test/battle/ability/emergency_exit.c]: fixed

File: test/battle/ability/emergency_exit.c [hunk 2]
```diff
@@ -8,31 +8,42 @@ SINGLE_BATTLE_TEST("Emergency Exit switches out when taking 50% max-hp damage")
...
+        TURN { MOVE(player, MOVE_SUPER_FANG); SEND_OUT(opponent, 1); }
     } SCENE {
         ANIMATION(ANIM_TYPE_MOVE, MOVE_SUPER_FANG, player);
         HP_BAR(opponent);
         ABILITY_POPUP(opponent, ABILITY_EMERGENCY_EXIT);
     }
 }
 
-SINGLE_BATTLE_TEST("Emergency Exit switches out when taking 50% max-hp damage after a restore hp hold effect was used")
+SINGLE_BATTLE_TEST("Emergency Exit does not switch out when taking 50% max-hp damage after a restore hp hold effect was used")
```
Comment (AsparagusEduardo) [inline test/battle/ability/emergency_exit.c:19]:
```suggestion
SINGLE_BATTLE_TEST("Emergency Exit does not switch out when going below 50% max-HP but healed via held item back above the threshold")
```

File: test/battle/ability/emergency_exit.c [hunk 3]
```diff
@@ -8,31 +8,42 @@ SINGLE_BATTLE_TEST("Emergency Exit switches out when taking 50% max-hp damage")
...
+        TURN { MOVE(player, MOVE_SUPER_FANG); }
     } SCENE {
         ANIMATION(ANIM_TYPE_MOVE, MOVE_SUPER_FANG, player);
         HP_BAR(opponent);
         ANIMATION(ANIM_TYPE_GENERAL, B_ANIM_HELD_ITEM_EFFECT, opponent);
         NOT ABILITY_POPUP(opponent, ABILITY_EMERGENCY_EXIT);
     }
 }
+
+SINGLE_BATTLE_TEST("Emergency Exit switches out when taking 50% max-hp damage after a restore hp hold effect was used")
```
Comment (AsparagusEduardo) [inline test/battle/ability/emergency_exit.c:35]:
```suggestion
SINGLE_BATTLE_TEST("Emergency Exit switches out when going below 50% max-HP but healing via held item is not enough to go back above the threshold")
```

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]
Re-wrote my name proposal bc I think I was very sleepy for it to make much sense ^^;
Besides that, lgtm

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**